puma.rb

```diff
@@ -6,12 +6,14 @@
   respawn 'yes'
   pid_file (path / 'tmp/pids/puma.pid').abs
 
-  command (path / 'bin/puma').abs
+  puma_path = (path / 'bin/puma').abs
+  socket_path = (path / "tmp/sockets/puma.socket").abs
```

Comment:
I agree on fancypath, but also with sticking to the babushka way for consistency in the short term